crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -195,13 +200,14 @@ impl BuildSpecCommand {
 		let para_id = self.id.unwrap_or(DEFAULT_PARA_ID);
 		// Notify user in case we need to build the parachain project.
 		if !self.release {
-			cli.warning("NOTE: this command defaults to DEBUG builds for development chain types. Please use `--release` (or simply `-r` for a release build...)")?;
+			cli.warning(
+				"NOTE: this command defaults to DEBUG builds for development chain types. Please use `--release` (or simply `-r` for a release build...)",
+			)?;
 			#[cfg(not(test))]
 			sleep(Duration::from_secs(3))
 		}
 
-		let spinner = cliclack::spinner();
-		spinner.start("Generating chain specification...");
+		cli.info("Generating chain specification...")?;
```

Comment:
Why all the info outputs instead of the spinner? I'd imagine its just going to flood the terminal with messages which will just be annoying after the first time.

Reply:
It was to fix the previous logs: https://github.com/r0gue-io/pop-cli/pull/334